common/setups/rasr/config/am_config.py

```diff
@@ -0,0 +1,77 @@
+__all__ = ["TransitionTdp", "SilenceTdp", "AmRasrConfig"]
+
+from dataclasses import dataclass
+from typing import Optional, Tuple, Union
+
+from sisyphus import tk
+
+import i6_core.am as am
+import i6_core.rasr as rasr
+
+
+@dataclass()
+class _Tdp:
+    loop: Union[float, str]
+    forward: Union[float, str]
+    skip: Union[float, str]
+    exit: Union[float, str]
+
+    def get_tuple(
+        self,
+    ) -> Tuple[
+        Union[float, str], Union[float, str], Union[float, str], Union[float, str]
+    ]:
+        return self.loop, self.forward, self.skip, self.exit
+
+
+@dataclass()
+class TransitionTdp(_Tdp):
+    loop: Union[float, str] = 3.0
+    forward: Union[float, str] = 0.0
+    skip: Union[float, str] = "infinity"
+    exit: Union[float, str] = 0.0
+
+
+@dataclass()
+class SilenceTdp(_Tdp):
+    loop: Union[float, str] = 0.0
+    forward: Union[float, str] = 3.0
+    skip: Union[float, str] = "infinity"
+    exit: Union[float, str] = 20.0
```

Comment:
does it make sense to define this as separate dataclasses? I would say these are just specific instances of the dataclass `_Tdp` that are by chance used as default value here (but Tina would use different values)

additionally `TransitionTdp` -> `SpeechTdp`